src/Views/users/login.phtml

```diff
@@ -0,0 +1,20 @@
+<?php
+$title = 'login';
+?>
+
+<div class="outerDiv text-center p-4 m-5 mx-auto">
+
+    <form method="post"></form>
+            <div class="mb-3">
+                <div id="loginIntro" class="form-text">Log into the marketplace</div>
+                <label for=inputEmail" class="form-label"> Email Address:</label>
+                <input type="email" class="form-control" name="InputEmail" aria-describedby="emailHelp" placeholder="example@123.co.uk">
```

Comment:
- Element names should be in camelCase instead of PascalCase.
- Use example.com when an example domain name is needed.